<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassphraseDialog</class>
 <widget class="QDialog" name="PassphraseDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter passphrase</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-2</number>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="OKBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="PassphraseText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Passphrase:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passphraseLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKBox</sender>
   <signal>accepted()</signal>
   <receiver>PassphraseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKBox</sender>
   <signal>rejected()</signal>
   <receiver>PassphraseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
